src/main/java/com/documaster/rms/noark/ws/noarkentities/Posisjon.java

```diff
@@ -24,6 +24,7 @@ public Posisjon(String koordinatsystem, String koordX, String koordY, String koo
 		this.koordX = koordX;
 		this.koordY = koordY;
 		this.koordZ = koordZ;
+		serializeKoordZ = true;
```

Comment:
We could invoke the setter instead as it would take care of this itself.

I don't mind the current implementation either though. Just a suggestion.

Reply:
Fixed with https://github.com/documaster/ws-client-n5-java/pull/56/commits/03708d709880a9c50d0a71d2b3bcdc68cea83989